README.md

```diff
@@ -1,11 +1,127 @@
-# Ursim suite of tools
+<!-- omit from toc -->
+# Simulation suite for the UR robot family
 
-Here are few examples of all the possible uses of ursim, the official UR robos controller simulator [[win](https://www.universal-robots.com/download/software-ur20ur30/simulator-non-linux/offline-simulator-e-series-and-ur20ur30-ur-sim-for-non-linux-5161/)],[[linux](https://www.universal-robots.com/download/software-ur20ur30/simulator-linux/offline-simulator-e-series-and-ur20ur30-ur-sim-for-linux-5161/)].
+<!-- omit from toc -->
+###### SAM XL COLLECTION OF TOOLS TO SIMULATE AND CONTROL UR ROBOTS VIRTUAL TWINS OFF-LINE
 
+<!-- omit from toc -->
+##### Table of content
 
-Docker compose is used for the setup, pulling the official images from Universal Robots and RoboDK. If you intend to use on your real robot, please make sure you use the correct [official ursim tag](https://hub.docker.com/r/universalrobots/ursim_e-series/tags) corresponding to you phisical controller Polyscope version. 
+- [Introduction](#introduction)
+- [Installation of the virtual robot controller](#installation-of-the-virtual-robot-controller)
+  - [Virtual machine method](#virtual-machine-method)
+  - [Docker method](#docker-method)
+    - [Install Docker on Windows](#install-docker-on-windows)
+    - [Install Docker on macOS](#install-docker-on-macos)
+    - [Install Docker on Linux](#install-docker-on-linux)
+  - [RoboDK installation](#robodk-installation)
+- [Move and program the ursim virtual controller](#move-and-program-the-ursim-virtual-controller)
+- [Move and program the RoboDK virtual twin](#move-and-program-the-robodk-virtual-twin)
+- [Appendix](#appendix)
+  - [File provided for the Docker installation method](#file-provided-for-the-docker-installation-method)
+- [Additional features](#additional-features)
+- [References](#references)
+
+
+## Introduction
+
+Simulating a robot and performing off-line programming are crucial aspects to de-risk the success of your application. It allows you to test and validate your robot's behaviour in a virtual environment, reducing the time to debug and detect errors, which can take a long time on a real robot!
+
+This repository contains several tools and instructions on how to use Universal Robots (UR) offline simulator (ursim), a virtual controller replica of a real robot brain, capable of replicating the behaviour of a real robot one-to-one, with all features and programming capabilities of a real robot, minus the physical body of the arm!
```

Comment:
There are no brains here, just controllers. I'd suggest not using words like *brain*.